controllers/integration/integration_adapter.go

```diff
@@ -489,7 +494,7 @@ func (a *Adapter) EnsureApplicationSnapshotEnvironmentBindingExist() (results.Op
 }
 
 func (a *Adapter) getApplicationSnapshotComponents(components []hasv1alpha1.Component) *[]appstudioshared.BindingComponent {
-	bindingComponents := []appstudioshared.BindingComponent{}
+	var bindingComponents []appstudioshared.BindingComponent
```

Comment:
Can you please make clear - any problem with the origin way?  Or, it's recommended to use the changed way in some condition? Thanks

Reply:
I applied the suggested approach after I got a warning from the IDE about `Empty slice declaration using a literal`. After some digging, I've found that this warning can be ignored ([jetbrains ticket here](https://youtrack.jetbrains.com/issue/GO-5317)), so I've decided to revert these changes to use your original approach.

Reply:
Thanks for your answer!

Reply:
@jinqi7 I reverted the mentioned changes, please take a look

Reply:
OK.